tools/release.js

```diff
@@ -62,26 +74,34 @@ async function startRelease() {
 
   // ---------------------------------------------------------------------------
   log(`Step 4/${STEPS}: creating git commit...`);
+  // For the commit, we make sure prettier has been run.
```

Comment:
i think the prettier check should be done as a separate step (probably one of the first)